lib/core/elastic_connector_actions.rb

```diff
@@ -52,6 +52,10 @@ def get_connector(connector_id)
         client.get(:index => Utility::Constants::CONNECTORS_INDEX, :id => connector_id, :ignore => 404).with_indifferent_access
       end
 
+      def get_job(job_id)
+        client.get(:index => Utility::Constants::JOB_INDEX, :id => job_id, :ignore => 404).with_indifferent_access
```

Comment:
Do we really need to use `with_indifferent_access` here?

Reply:
Actually, it does. 

See my [slides](https://docs.google.com/presentation/d/1EKSoFGIvIATWpP2kN9WC7nzmeFQD_D6ukmUNOPkVsWc/edit#slide=id.g16453f538f0_0_0)